_includes/tutorials/versioned-ktables/kstreams/markup/test/make-test-dir.adoc

```diff
@@ -0,0 +1,5 @@
+First create a directory for the tests to live in:
```

Comment:
comma after "First"? to be consistent with the next file. Though idk how big a can of worms this opens up (not sure we're consistent across KT)